src/quo2/components/buttons/wallet_ctas/style.cljs

```diff
@@ -4,7 +4,10 @@
   {:padding-top        24
    :padding-bottom     12
    :padding-horizontal 20
-   :flex-direction     :row})
+   :flex-direction     :row
+   :justify-content    :center
+   :flex               1
+   :max-height         106})
```

Comment:
hmm, should we maybe pass a container style prop and set the max height on the use in the page it's currently used on? 🤔

Reply:
@J-Son89 why? The height is always 106

Reply:
Just a suggestion as I was unsure of why this wasn't there before - assumed you added this to make it work on the page where it is being used. Either way is fine 👍